src/index.js

```diff
@@ -24,26 +24,15 @@ function render(
   const baseElement = customBaseElement || customContainer || document.body
   const container = customContainer || baseElement.appendChild(div)
 
-  const plugins = mountOptions.global?.plugins || []
-
-  if (store) {
-    const {createStore} = require('vuex')
-    plugins.push(createStore(store))
-  }
-
-  if (routes) {
-    const requiredRouter = require('vue-router')
-    const {createRouter, createWebHistory} =
-      requiredRouter.default || requiredRouter
-
-    const routerPlugin = createRouter({history: createWebHistory(), routes})
-    plugins.push(routerPlugin)
+  if (store || routes) {
+    console.warn(`Providing 'store' or 'routes' options is no longer available.
+You need to create a router/vuex instance and provide it through 'global.plugins'.
+Check out the test examples on GitHub for further details.`)
   }
 
   const wrapper = mount(Component, {
     ...mountOptions,
```

Comment:
https://testing-library.com/docs/vue-testing-library/api/#options

in the docs it says `props` is supported, but it doesn't look like it here, but maybe I'm missing something? Also doesn't seem like `baseElement` and `container` are _not_ shown in the docs?

Reply:
Hi! 

> in the docs it says props is supported, but it doesn't look like it here, but maybe I'm missing something?
Docs are still meant to match VTL for Vue 2. This is why it explicitly says that `props` is added. In Vue 3, Vue Test 

Utils already accepts a `prop` key so there's nothing to do on our side.

> Also doesn't seem like baseElement and container are not shown in the docs?

Yep, that's true. Fancy to open up a PR? 😉  We could use [RTL docs](https://testing-library.com/docs/react-testing-library/api#render-options) as example.

_btw, if you have further questions regarding VTL for Vue 3, feel free to open up new issues so this PR keeps focused on router/vuex :)_

Reply:
@afontcu https://github.com/testing-library/testing-library-docs/pull/767 done!